README.md

```diff
@@ -61,3 +61,8 @@ Feel free to improve this list using GitHub pull request mechanism, see [CONTRIB
 * [Q# Advent Calendar 2018](https://blogs.msdn.microsoft.com/visualstudio/2018/11/15/q-advent-calendar-2018/) - a collection of Q#-related blog posts
 * [The Hitchhiker’s Guide to the Quantum Computing and Q#](https://blogs.msdn.microsoft.com/uk_faculty_connection/2018/02/26/the-hitchhikers-guide-to-the-quantum-computing-and-q-blog/)
 * El Bruno [blog posts](https://elbruno.com/tag/q/) and [examples](https://github.com/elbruno/Blog/search?utf8=✓&q=extension%3Aqs&type=)
+
+## License
+[![CC0](http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg)](https://creativecommons.org/publicdomain/zero/1.0/)
+
+To the extent possible under law, [Sindre Sorhus](https://sindresorhus.com) has waived all copyright and related or neighboring rights to this work.
```

Comment:
Uhm, it should not be my name here, but rather the maintainer's name.

Reply:
Oops...